prometheus/src/main/scala/fr/davit/akka/http/metrics/prometheus/PrometheusSettings.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019 Michel Davit
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.davit.akka.http.metrics.prometheus
+
+/**
+  * Settings for Prometheus metrics
+  *
+  * @param durationBuckets     Buckets to use for calculating request duration histogram (seconds).
+  * @param requestSizeBuckets  Buckets to use for calculating request size histogram (bytes).
+  * @param responseSizeBuckets Buckets to use for calculating response size histogram (bytes).
+  */
+case class PrometheusSettings(
+    durationBuckets: Seq[Double] = Seq(0.005, 0.01, 0.025, 0.05, 0.1, 0.25, 0.5, 1, 2, 5, 10),
+    requestSizeBuckets: Seq[Double] = Seq(0, 1024, 10240, 102400, 1048576, 10485760),
+    responseSizeBuckets: Seq[Double] = Seq(0, 1024, 10240, 102400, 1048576, 10485760)
```

Comment:
I think 0 is not useful since there won't be anything in the 1st bucket.

I've tried to look for sensible defaults. I only could find this:
```go
// define response size buckets (bytes)
var ResponseSizeBuckets = append(append(append(append(
	prometheus.LinearBuckets(100, 100, 5),
	prometheus.LinearBuckets(1000, 1000, 5)...),
	prometheus.LinearBuckets(10000, 10000, 5)...),
	prometheus.LinearBuckets(1000000, 1000000, 5)...),
)
```

So i'll probably start with 128 bytes and adopt a more exponential scaling

Reply:
I tried to provide a short list of values for the request/response buckets that cover a large range of values. I've added a few values in the lower end, was that what you needed?

Regarding the "0" in the response size: I've removed it for now, but generally there can also be empty responses (there's also HTTP 204 specifically for that).